service/src/main/resources/db/migration/R__message_account_access_view.sql

```diff
@@ -1,22 +1,29 @@
 DROP VIEW IF EXISTS message_account_access_view;
 
-CREATE VIEW message_account_access_view(employee_id, account_id) AS (
-    SELECT employee.id AS employee_id, acc.id AS account_id
+CREATE VIEW message_account_access_view(account_id, employee_id, person_id) AS (
```

Comment:
I'd rather leave this view as it is (or maybe rename it to `employee_account_access_view`) and just check for person access directly in the queries. Or maybe create a separate view for citizen account access. There's no use-case where we lookup with both ids at the same time and this implementation has performance implications.